packages/groqd/test-utils/runQuery.ts

```diff
@@ -3,8 +3,9 @@ import { evaluate, parse } from "groq-js";
 import { pokemonDataset } from "./pokemon";
 import { BaseQuery } from "../src/baseQuery";
 import { userDataset } from "./users";
+import { GroqdParseError, makeSafeQueryRunner } from "../src";
```

Comment:
Changes here are just to use our external-facing `makeSafeQueryRunner` as part of the test running utils.